src/tiled/tilesetdocument.cpp

```diff
@@ -76,6 +76,8 @@ TilesetDocument::TilesetDocument(const SharedTileset &tileset)
     connect(this, &TilesetDocument::propertiesChanged,
             this, &TilesetDocument::onPropertiesChanged);
 
+    // TODO: add components?
```

Comment:
Properties are handled here because of tile property inheritance. I think components will only need to be added when they are also inherited.